api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/ModelLoaderTest.kt

```diff
@@ -0,0 +1,107 @@
+package org.jetbrains.kotlinx.dl.api.inference.keras
+
+import org.jetbrains.kotlinx.dl.api.core.initializer.GlorotNormal
+import org.jetbrains.kotlinx.dl.api.core.initializer.GlorotUniform
+import org.jetbrains.kotlinx.dl.api.core.initializer.HeNormal
+import org.jetbrains.kotlinx.dl.api.core.initializer.HeUniform
+import org.jetbrains.kotlinx.dl.api.core.initializer.Identity
+import org.jetbrains.kotlinx.dl.api.core.initializer.LeCunNormal
+import org.jetbrains.kotlinx.dl.api.core.initializer.LeCunUniform
+import org.jetbrains.kotlinx.dl.api.core.initializer.RandomNormal
+import org.jetbrains.kotlinx.dl.api.core.initializer.RandomUniform
+import org.jetbrains.kotlinx.dl.api.core.initializer.TruncatedNormal
+import org.jetbrains.kotlinx.dl.api.core.layer.core.Dense
+import org.jetbrains.kotlinx.dl.api.inference.savedmodel.SavedModel
+import org.junit.jupiter.api.Test
+import java.io.File
+import org.junit.jupiter.api.Assertions.assertArrayEquals
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.assertAll
+
+private const val INITALIZER_PATH = "src/test/resources/inference/keras/ModelLoader/initializer_identity.json"
+
+class ModelLoaderTest {
+    @Test
+    fun `load initializers from file`(){
```

Comment:
It's an interesting idea, I did the close thing but via the whole model with weights.
Is this model config from Keras? If yes, could please add in the resources folder (with python subfolder) the Jupyter Notebook to reproduce it?

Reply:
I added the notebook.

The JSON is an export of this model:
```python
model = keras.Sequential(
    [
        keras.Input((32,32,)), 
        keras.layers.Dense(32, kernel_initializer=keras.initializers.GlorotUniform(), name='GlorotUniform'),
        keras.layers.Dense(32, kernel_initializer=keras.initializers.GlorotNormal(), name='GlorotNormal'),
        keras.layers.Dense(32, kernel_initializer=keras.initializers.he_normal(), name='HeNormal'),
        keras.layers.Dense(32, kernel_initializer=keras.initializers.he_uniform(), name='HeUniform'),
        keras.layers.Dense(32, kernel_initializer=keras.initializers.lecun_normal(), name='LeCunNormal'),
        keras.layers.Dense(32, kernel_initializer=keras.initializers.lecun_uniform(), name='LeCunUniform'),
        keras.layers.Dense(32, kernel_initializer=keras.initializers.Zeros(), name='Zeros'), 
        keras.layers.Dense(32, kernel_initializer=keras.initializers.Constant(value=2), name='Constant'),
        keras.layers.Dense(32, kernel_initializer=keras.initializers.Ones(), name='Ones'),
        keras.layers.Dense(32, kernel_initializer=keras.initializers.RandomNormal(mean = 0.,  stddev = 1.), name='RandomNormal'),
        keras.layers.Dense(32, kernel_initializer=keras.initializers.RandomUniform(minval = -0.05, maxval = 0.05), name='RandomUniform'),
        keras.layers.Dense(32, kernel_initializer=keras.initializers.TruncatedNormal(), name='TruncatedNormal'),
        keras.layers.Dense(32, kernel_initializer=keras.initializers.Identity(gain = 3.4), name='Identity')
])
```